lib/std/Build/Step.zig

```diff
@@ -39,7 +39,7 @@ test_results: TestResults,
 
 /// The return address associated with creation of this step that can be useful
 /// to print along with debugging messages.
-debug_stack_trace: [n_debug_stack_frames]usize,
+debug_stack_trace: ?[]usize,
```

Comment:
why does it need to be optional now?

Reply:
It doesn't, I had this to convey intent. I'll remove it if it's not idiomatic.

Reply:
With it being an optional, it's unclear what's the difference between `null` and being a length 0 stack trace. I think there is no meaningful difference, therefore it should be non-optional.